src/m365/spo/commands/homesite/homesite-remove.spec.ts

```diff
@@ -144,10 +215,59 @@ describe(commands.HOMESITE_REMOVE, () => {
     });
 
     await command.action(logger, { options: { force: true } });
-    assert(homeSiteRemoveCallIssued);
+    assert(postStub.calledOnce);
+    assert.deepStrictEqual(postStub.lastCall.args[0].url, "https://contoso-admin.sharepoint.com/_api/SPO.Tenant/RemoveTargetedSite");
+  });
+
+  it('removes the Home Site specified by URL', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://contoso-admin.sharepoint.com/_api/SPO.Tenant/GetTargetedSitesDetails`) {
+        return homeSites;
+      }
+
+      throw 'Invalid request';
+    });
+
+    sinon.stub(spo, 'getSiteAdminPropertiesByUrl').resolves({ SiteId: siteId } as any);
+    const postStub = sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `https://contoso-admin.sharepoint.com/_api/SPO.Tenant/RemoveTargetedSite` &&
+        opts.data?.siteId === siteId) {
```

Comment:
Let's not check the site ID like this.
Better is to use an assert statement to check the body of the fired request like we do in other commands.